### PR TITLE
fix/broker package end date

### PR DESCRIPTION
**Pull request by mugnivenko**

### CODE REVIEW

File: components/HackneyDS/DatePicker/index.js [hunk 1]
```diff
@@ -188,14 +212,18 @@ const DatePicker = ({
           )}
         </div>
       )}
-      {hasClear && date && <div className='date-picker__additional-action clear-datepicker' onClick={clearDate}><CrossIcon /></div>}
+      {hasClear && date && (
```
Comment (akeamwoodsND) [inline components/HackneyDS/DatePicker/index.js:215]: rename 'hasClear' to 'isClear'

**Review by akeamwoodsND — COMMENTED** [review]
There's a lot going on in this PR which isn't to do with the broker package from date https://app.clickup.com/2400535/ why is this?

File: components/HackneyDS/DatePicker/index.js (hunk 1 above)
Comment (mugnivenko) [reply components/HackneyDS/DatePicker/index.js:215]: changed to "hasClearButton"

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Have opacity lowered on the date picker component **when** it is disabled (e.g. ongoing is checked)
Screenshot 2021-11-02 at 15 27 58

**Comment by akeamwoodsND** [comment]
> Have opacity lowered on the date picker component **when** it is disabled (e.g. ongoing is checked) Screenshot 2021-11-02 at 15 27 58

Screenshot 2021-11-02 at 15 29 15

File: api/SWR/suppliers.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 
-export const useSuppliers = ({ params, supplierId, shouldFetch }) => {
-  const supplierIdUrl = supplierId ? `/${supplierId}` : '';
-
-  return useFetchWithParams({
+export const useSuppliers = ({ params, shouldFetch }) =>
+  useFetchWithParams({
     params,
     shouldFetch,
-    url: `/suppliers${supplierIdUrl}`
```
Comment (akeamwoodsND) [inline api/SWR/suppliers.js:9]: why have we changed this?

File: components/HackneyDS/DatePicker/DatePicker.stories.js [hunk 3]
```diff
@@ -61,4 +61,27 @@ LabelHintError.args = {
...
+
+export const WithClearButton = Template.bind({});
+WithClearButton.args = {
+  formId: 'label-hint-error',
+  hint: 'Date picker hint',
+  label: 'Date picker label',
+  day: { label: 'Day' },
+  month: { label: 'Month' },
+  year: { error: 'Error year', label: 'Year' },
+  hasClear: true,
```
Comment (akeamwoodsND) [inline components/HackneyDS/DatePicker/DatePicker.stories.js:74]: wrong prop name

File: components/HackneyDS/DatePicker/DatePicker.stories.js [hunk 4]
```diff
@@ -61,4 +61,27 @@ LabelHintError.args = {
...
+export const Disabled = Template.bind({});
+Disabled.args = {
+  formId: 'label-hint-error',
+  hint: 'Date picker hint',
+  label: 'Date picker label',
+  day: { label: 'Day' },
+  month: { label: 'Month' },
+  year: { error: 'Error year', label: 'Year' },
+  disabled: true,
+  hasClear: true,
```
Comment (akeamwoodsND) [inline components/HackneyDS/DatePicker/DatePicker.stories.js:86]: wrong name

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Background on this icon should not go grey on hover:
Screenshot 2021-11-02 at 15 47 28

File: api/SWR/suppliers.js (hunk 2 above)
Comment (mugnivenko) [reply api/SWR/suppliers.js:9]: i create separate function for request without params

File: components/Pages/CarePackages/BrokerageCost.js [hunk 5]
```diff
@@ -65,7 +64,7 @@ const BrokerageCost = ({
                     value={item.cost}
                     onChangeValue={(value) => changeNeed(getter, setter, 'cost', value, index)}
                   />
-                  <Container className="brokerage__cost-add-need" display="flex">
+                  <Container width='' className="brokerage__cost-add-need" display="flex">
```
Comment (akeamwoodsND) [inline components/Pages/CarePackages/BrokerageCost.js:67]: width='' ?

**Review by akeamwoodsND — APPROVED** [review]

**Pull request merged**